README.md

```diff
@@ -13,6 +13,17 @@ on your docker machine.
 ## Development
 This rails app uses the [dotenv gem](https://github.com/bkeepers/dotenv). See `.env.example`
 
+These steps will run the server for development:
+1. Install Ruby
+2. Install bundler
+3. Install and run database
+4. Create an .env file from .env.example
+5. Create a secrets.yml from secrets.yml.example
+6. `bundle exec rake db:create RAILS_ENV=test`
+7. `bundle exec rake db:schema:load RAILS_ENV=test`
```

Comment:
I think steps 6 & 7 can be replaced with `rake db:test:prepare`

Reply:
👍